Source/WebKit/WebProcess/GPU/graphics/RemoteRenderingBackendProxy.h

```diff
@@ -65,17 +65,13 @@ namespace WebKit {
 class WebPage;
 
 class RemoteRenderingBackendProxy
-    : private IPC::MessageReceiver
-    , public GPUProcessConnection::Client {
+    : public IPC::Connection::Client {
```

Comment:
maybe private..

Reply:
We need this to be public so that WeakPtr<RemoteRenderingBackendProxy> can be created (previously CanMakeWeakPtr was available through the public GPUProcessConnection::Client base).